### DISCUSSION TITLE
macOS Docker Desktop Getting Bash by Default for Terminal

### DISCUSSION THREAD

**Discussion by ddspell** [BretFisher/ama#221] [Q&A]
My Dockerfile has SHELL set to /bin/bash but when the terminal loads, it uses the basic shell.
Where is the setting on macOS to define the default shell used by Docker Desktop terminal?

**Comment by BretFisher** [comment]
My bet is you need to make iTerm your default macOS terminal https://stackoverflow.com/questions/60210024/how-can-i-use-iterm-as-default-terminal-on-macos

**Comment by ddspell** [comment]
I made the change. Restarted iTerm and Docker Desktop. 
Same result except now the menu option in iTerm to "Make iTerm2 Default Term" is grayed out, so at least that confirms that changed.

Here's what I see in the iTerm window opened by Docker Desktop...
```
> docker exec -it 127f97e0deffb24767bf1a6346db720bdc634025dfc46f1331d51a5d2ab38050 /bin/sh
# echo $SHELL
/bin/bash
# ps -p $$
  PID TTY          TIME CMD
   14 pts/0    00:00:00 sh
#
```

**Reply by ddspell** [reply]
Right after I posted that, I think I see the issue.
It specifies the shell in the command line. Duh!

OK, so where is that set in Docker Desktop?

**Reply by BretFisher** [reply]
Sorry, this isn't really a troubleshooting repo. I recommend https://github.com/docker/for-mac/issues
